test/gen_inputs.py

```diff
@@ -0,0 +1,65 @@
+"""Generates subsected inputs to allow for fast testable builds.
+
+
+Takes input release files (LOINC, SNOMED, etc) and subsects them. Then, a build will be run and outputs of that will be
+test inputs.
+
+TODO: add cli
+ - cache option
+ - should be able to subesct or gen test outputs independently
+"""
+try:
+    # noinspection PyUnresolvedReferences pycharm_confused_by_test_root
+    from test.config import TEST_IN_DIR_SUBSET as OUTDIR, BUILD_CONFIG_PATH, SUBSET_BUILD_GEN_INPUTS
+    from loinclib import Configuration
+except ModuleNotFoundError:
+    from config import TEST_IN_DIR_SUBSET as OUTDIR, BUILD_CONFIG_PATH, SUBSET_BUILD_GEN_INPUTS
+    from loinclib import Configuration
+
+
+# TODO: make dirs if don't exist
+#  - all subset dirs. do that here at top of file, or maybe in each func
+
+
+def run_subsect_build():
+    """For outputs to be stored after running build on subsected release files"""
+    # TODO: How to pass alternative config path?
+    #  - is that a CLI option?
+    #    - if not, a. add one, b. temporarily replace config at root. | 'a' seems much better
+    print()
+
+
+def gen_subsect_config():
```

Comment:
## Test build config
Will need to generate an alternate config for these subsected files. Then when running build, need to make sure it uses that.